#! /bin/sh /usr/share/dpatch/dpatch-run
## 03manpages-more-fixes.dpatch by Manuel Prinz <debian@pinguinkiste.de>
##
## DP: More fixes of lintian warnings.

@DPATCH@
diff -urNad openmpi-1.2.6~/ompi/mpi/man/man3/MPI_Bcast.3 openmpi-1.2.6/ompi/mpi/man/man3/MPI_Bcast.3
--- openmpi-1.2.6~/ompi/mpi/man/man3/MPI_Bcast.3	2008-06-01 18:28:02.000000000 +0200
+++ openmpi-1.2.6/ompi/mpi/man/man3/MPI_Bcast.3	2008-06-01 19:10:15.000000000 +0200
@@ -26,7 +26,7 @@
 	const MPI::Datatype& \fIdatatype\fP, int \fIroot\fP) const = 0
 
 .SH INPUT/OUTPUT PARAMETERS
-.fR
+.ft R
 .TP 1i
 buffer
 Starting address of buffer (choice).
diff -urNad openmpi-1.2.6~/ompi/mpi/man/man3/MPI_Type_create_darray.3 openmpi-1.2.6/ompi/mpi/man/man3/MPI_Type_create_darray.3
--- openmpi-1.2.6~/ompi/mpi/man/man3/MPI_Type_create_darray.3	2008-06-01 18:28:02.000000000 +0200
+++ openmpi-1.2.6/ompi/mpi/man/man3/MPI_Type_create_darray.3	2008-06-01 19:04:54.000000000 +0200
@@ -98,7 +98,7 @@
 - MPI_DISTRIBUTE_NONE - Dimension not distributed. 
 .fi
 .sp
-The constant MPI_DISTRIBUTE_DFLT_DARG specifies a default distribution argument. The distribution argument for a dimension that is not distributed is ignored. For any dimension \fIi\fP in which the distribution is MPI_DISTRIBUTE_BLOCK, it erroneous to specify \fIarray_of_dargs[i]\fP \fI\*\fP \fIarray_of_psizes[i]\fP < \fIarray_of_gsizes[i]\fP.
+The constant MPI_DISTRIBUTE_DFLT_DARG specifies a default distribution argument. The distribution argument for a dimension that is not distributed is ignored. For any dimension \fIi\fP in which the distribution is MPI_DISTRIBUTE_BLOCK, it erroneous to specify \fIarray_of_dargs[i]\fP \fI*\fP \fIarray_of_psizes[i]\fP < \fIarray_of_gsizes[i]\fP.
 .sp
 For example, the HPF layout ARRAY(CYCLIC(15)) corresponds to MPI_DISTRIBUTE_CYCLIC with a distribution argument of 15, and the HPF layout ARRAY(BLOCK) corresponds to MPI_DISTRIBUTE_BLOCK with a distribution argument of MPI_DISTRIBUTE_DFLT_DARG. 
 .sp
diff -urNad openmpi-1.2.6~/ompi/mpi/man/man3/MPI_Wait.3 openmpi-1.2.6/ompi/mpi/man/man3/MPI_Wait.3
--- openmpi-1.2.6~/ompi/mpi/man/man3/MPI_Wait.3	2008-06-01 18:28:02.000000000 +0200
+++ openmpi-1.2.6/ompi/mpi/man/man3/MPI_Wait.3	2008-06-01 19:02:02.000000000 +0200
@@ -50,7 +50,7 @@
 .sp
 One is allowed to call MPI_Wait with a null or inactive request argument. In this case the operation returns immediately with empty status. 
 
-.NOTES
+.SH NOTES
 Successful return of MPI_Wait after an MPI_Ibsend implies that the user send buffer can be reused  i.e., data has been sent out or copied into a buffer attached with MPI_Buffer_attach. Note that, at this point, we can no longer cancel the send (for more information, see Section 3.8 of the MPI-1 Standard, "Probe and Cancel"). If a matching receive is never posted, then the buffer cannot be freed. This runs somewhat counter to the stated goal of MPI_Cancel (always being able to free program space that was committed to the communication subsystem). 
 .sp
 Example: Simple usage of nonblocking operations and  MPI_Wait. 
@@ -66,7 +66,6 @@
         **** do some computation **** 
         CALL MPI_WAIT(request, status, ierr) 
     END IF 
-.ni
 
 .SH ERRORS
 Almost all MPI routines return an error value; C routines as the value of the function and Fortran routines in the last argument. C++ functions do not return errors. If the default error handler is set to MPI::ERRORS_THROW_EXCEPTIONS, then on error the C++ exception mechanism will be used to throw an MPI::Exception object.
diff -urNad openmpi-1.2.6~/orte/tools/orterun/orterun.1 openmpi-1.2.6/orte/tools/orterun/orterun.1
--- openmpi-1.2.6~/orte/tools/orterun/orterun.1	2008-03-05 22:48:25.000000000 +0100
+++ openmpi-1.2.6/orte/tools/orterun/orterun.1	2008-06-01 18:57:46.000000000 +0200
@@ -455,7 +455,7 @@
 .SS Specifying Hosts
 .
 Hosts can be specified in a number of ways. The most common of which is in a
-'hostfile' or 'machinefile'. If our hostfile contain the following information:
+\&'hostfile' or 'machinefile'. If our hostfile contain the following information:
 .
 .
 
